jep/1/README.adoc

```diff
@@ -546,6 +523,21 @@ If one or more contributors are not responding
 and the sponsor chooses to move forard without their feedback,
 they should document that choice in the "<<Reasoning>>" section of the JEP.
 
+==== Resolving Disputes
+
+Except for the decision by a reviewer to accept or reject a JEP,
+the JEP process is run by
+link:https://en.wikipedia.org/wiki/Consensus_decision-making[consensus].
+It is the responsibility of the every contributor to respect other contributors,
+listen to their perspectives, and attempt to find solution that work for everyone.
+If consensus cannot be achieved or if there is strong disagreement with the decision of a reviewer,
+a contributor may request that the Jenkins Governance Board intervene.
+The board will take up the matter at their next regular meeting and render a decision at the meeting following.
```

Comment:
Just to clarify, do you mean "Jenkins Governance Board" or "Jenkins Governance Meeting"? These are different entities. Regular && public meetings apply only to the latter one.